pubsub/bloom/filter.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package bloom
+
+import (
+	"errors"
+
+	"github.com/ava-labs/avalanchego/utils/bloom"
+)
+
+const bytesPerHash = 8
+
+var errMaxBytes = errors.New("too large")
+
+type Filter interface {
```

Comment:
I know that this inherently does an interface compliance check with `New` returning this interface but could we add a blank check at the top of this file?